README.md

```diff
@@ -1,8 +1,261 @@
-# RFC
+# Amethyst RFCs - [Active RFC List TBD]
 
-This is the repository where we work and keep track of the all RFC documents related to Amethyst.
-Having this repository allows people to participate in decision making by opening a pull request with a new document.
+[Amethyst RFCs]:
 
-## Creating a new RFC
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
 
-The specifics of the RFC process are not yet discussed, but will be documented in the first RFC.
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the Amethyst community and
+the [sub-team]s.
+
+The "RFC" (Request for Comments) process is intended to provide a consistent
+and controlled path for new features to enter the engine and associated
+libraries, so that all stakeholders can be confident about the direction in which
+the engine is evolving.
+
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening] Should we put a link to the Discord server here?
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [What the process is]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [RFC Postponement]
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+Amethyst, its component crates, or the RFC process itself. The definition of substantial
+is somewhat fluid, and to an extent set by the community. Some examples might include:
+
+  - A change to a different crate used for physics calculations
+  - Changes to how the editor communicates with the engine
+  - Significant protocol changes to the networking library
+  - Anything that has the potential to produce a substantial negative impact on users
+
+Some changes do not require an RFC:
+
+  - Rephrasing, reorganizing, refactoring, or otherwise "changing shape does
+    not change meaning".
+  - Additions that strictly improve objective, numerical quality criteria
+    (warning removal, speedup, better platform coverage, more parallelism, trap
+    more errors, etc.)
+  - Additions only likely to be _noticed by_ other developers-of-amethyst,
+    invisible to users-of-amethyst.
+  - More tests and benchmarks
+
```

Comment:
If you can define "very minor" for all potential users of Amethyst. Breaking changes, even minor ones, are usually considered an entirely different beast and require much more scrutiny. @amethyst/core @magnonellie @TimonPost @azriel91 thoughts?

Reply:
@jojolepro any more thoughts on this?

Reply:
I don't think we should include the minor breaking changes exception as it's always good to have a log of what's happening and the discussion around it. There's nothing preventing you from making just a very small RFC. That said, as we still are in pre 1.0 state, it might be difficult to keep up with it in the long run, but I think that's gonna evolve pretty organically if it ends up being the case.

Reply:
Calling this resolved, then.